tests/test_bookmarks.py

```diff
@@ -190,9 +191,13 @@ def test_run(self):
                 first_rec = first_sync_records.get(stream).get('messages')[0].get('data')
             first_rec_id = first_rec.get('id')
             first_rec_version = first_rec.get('version')
-            updated_record = self.client.update(stream, first_rec_id, first_rec_version)
+            if stream == 'inventories':
+                updated_record = self.client.update(stream, obj_id=None, version=None, obj=first_rec)
+            else:
+                updated_record = self.client.update(stream, first_rec_id, first_rec_version)
             assert len(updated_record) > 0, "Failed to update a {} record".format(stream)
-            assert len(updated_record) == 1, "Updated too many {} records".format(stream)
+            if stream != 'inventories':
+                assert len(updated_record) == 1, "Updated too many {} records".format(stream)
```

Comment:
Done

Reply:
I think we can simplify this to:
```
if stream == 'inventories':
    updated_record = self.client.update(stream, obj_id=None, version=None, obj=first_rec)
else:
    updated_record = self.client.update(stream, first_rec_id, first_rec_version)
    assert len(updated_record) == 1, "Updated too many {} records".format(stream)
assert len(updated_record) > 0, "Failed to update a {} record".format(stream)
```

Reply:
I like the ordering I have because if we fail to get a value returned for a standard stream, we will end up failing with `Updated too many <stream-name> streams` rather than the `Failed to update a <stream-name> record` message.

Reply:
Which is not the case